<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Karthick-TM" ignored="0" name="Exception_1" tests="10" failures="10" timestamp="14 Jan 2019 07:03:23 GMT" time="2.435" errors="0">
  <testcase name="API_Method" time="2.332" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.285" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.332" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.332" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.348" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.348" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.332" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.36" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.363" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
  <testcase name="API_Method" time="2.392" classname="com.edr.businesscomponents.EdrFlow1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.edr.businesscomponents.EdrFlow1.API_Method(EdrFlow1.java:43)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- API_Method -->
</testsuite> <!-- Exception_1 -->
